tutorials/notebooks/keras/ptq/example_keras_activation_threshold.ipynb

```diff
@@ -0,0 +1,866 @@
+      "metadata": {
+        "id": "ba0c6e55-d474-4dc3-9a43-44b736635998"
+      },
+      "outputs": [],
+      "source": [
+        "quantized_models_dict = {}\n",
+        "\n",
+        "for error_method, q_config in q_configs_dict.items():\n",
+        "    # Create a CoreConfig object with the current quantization configuration\n",
+        "    ptq_config = mct.core.CoreConfig(quantization_config=q_config)\n",
+        "\n",
+        "    # Perform MCT post-training quantization\n",
+        "    quantized_model, quantization_info = mct.ptq.keras_post_training_quantization_experimental(\n",
+        "        in_model=float_model,\n",
+        "        representative_data_gen=representative_dataset_gen,\n",
+        "        core_config=ptq_config,\n",
+        "        target_platform_capabilities=tpc\n",
+        "    )\n",
+        "\n",
+        "    # Update the dictionary to include the quantized model\n",
+        "    quantized_models_dict[error_method] = {\n",
+        "        \"quantization_config\": q_config,\n",
+        "        \"quantized_model\": quantized_model,\n",
+        "        \"quantization_info\": quantization_info\n",
+        "    }\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Threshold and Distribution Visulisation"
+      ],
+      "metadata": {
+        "id": "A8UHRsh2khM4"
+      },
+      "id": "A8UHRsh2khM4"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "To assist with understanding we will now plot the threshold and distributuion from two of Mobilenetv2's layers.\n",
+        "\n",
+        "MCT quantisation_info stores threshold data per layer. However, to see the distribution of the activations the model needs to be rebuilt upto and including the layer chosen for distribution visulisation.\n",
+        "\n",
+        "To do this we first need to list the layer names. With keras this can be done easily for the first 10 layes with the following."
+      ],
+      "metadata": {
+        "id": "Y-0QLWFJkpFV"
+      },
+      "id": "Y-0QLWFJkpFV"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "a22e6d68-c40f-40bf-ab74-ff453011aeac",
+      "metadata": {
+        "id": "a22e6d68-c40f-40bf-ab74-ff453011aeac"
+      },
+      "outputs": [],
+      "source": [
+        "for index, layer in enumerate(float_model.layers):\n",
+        "    if index < 10:\n",
+        "        print(layer.name)\n",
+        "    else:\n",
+        "        break"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "c38d28f3-c947-4c7c-aafa-e96cc3864277",
+      "metadata": {
+        "id": "c38d28f3-c947-4c7c-aafa-e96cc3864277"
+      },
+      "source": [
+        "First activation layer in model is 'Conv1_relu'.\n",
+        "\n",
+        "For this particular model expanded_conv_project_BN demonstrates the difference between the two error metrics so we will also use this.\n",
```

Comment:
How is this layer demonstrates it? why this layer? this part need more background because this name (expanded_conv_project_BN) doesn't say anything to someone who is not an expert on MobileNetV2 architecture (and even then, because it is kind of an inner name that we give to the layer when building the model in MCT).

Reply:
the layer was found through testing iv now pointed this out in the line. i have also added a couple of words on the significance of this particular layer though, Im unsure if its worth keeping in as i think it just adds confusion.

Reply:
I think the addition is good and helpful for the understanding of what you're demonstrating. I would advise to keep it.